drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/SummaryPageView.java

```diff
@@ -0,0 +1,106 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.guided.dtable.client.wizard.column.pages;
+
+import java.util.List;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.event.dom.client.ChangeEvent;
+import com.google.gwt.user.client.ui.CheckBox;
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.ListBox;
+import org.drools.workbench.screens.guided.dtable.client.wizard.column.plugins.commons.BaseDecisionTableColumnPlugin;
+import org.drools.workbench.screens.guided.dtable.client.wizard.column.plugins.commons.DecisionTableColumnPlugin;
+import org.jboss.errai.ui.shared.api.annotations.DataField;
+import org.jboss.errai.ui.shared.api.annotations.EventHandler;
+import org.jboss.errai.ui.shared.api.annotations.Templated;
+
+import static org.drools.workbench.screens.guided.dtable.client.wizard.column.pages.common.DecisionTableColumnViewUtils.getCurrentIndexFromList;
+
+@Dependent
+@Templated
+public class SummaryPageView extends Composite implements SummaryPage.View {
+
+    private SummaryPage page;
+
+    @Inject
+    @DataField("pluginsList")
+    private ListBox pluginsList;
+
+    @Inject
+    @DataField("includeAdvanced")
+    private CheckBox includeAdvanced;
+
+    @Override
+    public void init(final SummaryPage page) {
+        this.page = page;
+
+        setupPlugins();
+    }
+
+    @EventHandler("pluginsList")
+    public void onPluginSelected(ChangeEvent event) {
+        openSelectedPlugin();
+    }
+
+    @EventHandler("includeAdvanced")
+    public void onSelectIncludeAdvanced(final ChangeEvent event) {
+        page.setIncludeAdvanced(includeAdvanced.getValue());
+
+        setupPlugins();
+    }
+
+    private void setupPlugins() {
+        loadPluginList();
+        setupPluginList();
+    }
+
+    private void openSelectedPlugin() {
+        final String selectedValue = pluginsList.getSelectedValue();
+
+        page.openPage(selectedValue);
+    }
+
+    private void loadPluginList() {
+        final List<BaseDecisionTableColumnPlugin> plugins = page.pluginsByCategory();
+
+        pluginsList.clear();
+
+        for (final BaseDecisionTableColumnPlugin plugin : plugins) {
+            pluginsList.addItem(plugin.getTitle(),
+                                plugin.getIdentifier());
+        }
+    }
+
+    private void setupPluginList() {
+        pluginsList.setVisibleItemCount(pluginsList.getItemCount());
+        pluginsList.setSelectedIndex(selectedPlugin());
+    }
+
+    private int selectedPlugin() {
+        final DecisionTableColumnPlugin plugin = page.plugin();
+
+        if (plugin != null) {
+            return getCurrentIndexFromList(plugin.getIdentifier(),
+                                           pluginsList);
+        }
+
+        return -1;
```

Comment:
Selecting an element with `-1` would not raise any exception. I'm using this to "unselect" the element (keeping it clean).